### PR TITLE
Fixes livestream being "live" when it was really awaiting an upcoming livestream

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-api/src/data/live/data-source.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ export default class LiveStream extends RESTDataSource {
       null,
       { cacheOptions: { ttl: 60 } }
     );
-    return response.includes('og:video:url');
+
+    return (
+      response.includes('og:video:url') &&
+      !response.includes('\\"status\\":\\"LIVE_STREAM_OFFLINE\\"')
```
Comment (vinnyjth) [inline packages/apollos-api/src/data/live/data-source.js]: First issue - needed to check for this field

File: packages/apollos-api/src/data/live/data-source.js [hunk 2]
```diff
@@ -35,7 +39,7 @@ export default class LiveStream extends RESTDataSource {
     if (!channelId) return null;
 
     return {
-      isLive: this.getIsLive.bind(this, { channelId }),
+      isLive: await this.getIsLive.call(this, { channelId }),
```
Comment (vinnyjth) [inline packages/apollos-api/src/data/live/data-source.js]: Second issue - we were expecting a value when calling this function, but we were originally returning a bound function that when called, would resolve to an object containing an unresolved promise.
